### ISSUE TITLE
python 3.9 wheels

### ISSUE THREAD

**Issue by tekumara** [guildai/guildai#249]
I'm running macOS with brew installed python 3.9.

When trying to install guildai:
```
$ pip install guildai
ERROR: Could not find a version that satisfies the requirement guildai (from versions: none)
ERROR: No matching distribution found for guildai
```

Would be fantastic to have python 3.9 wheels.

**Comment by gar1t** [comment]
Yes for sure! 0.7.2 is going out today but we can make this a top priority for 0.7.3, which we can release quickly as a followup.

**Comment by gar1t** [comment]
These are available for a pre-release version of 0.7.3. You can install them using `pip install guildai --pre`.
